lib/private/DB/QueryBuilder/ExpressionBuilder/OCIExpressionBuilder.php

```diff
@@ -163,16 +163,16 @@ public function nonEmptyString($x) {
 	 *
 	 * @param string $column
 	 * @param mixed $type One of IQueryBuilder::PARAM_*
-	 * @return IQueryFunction
+	 * @return string
 	 */
-	public function castColumn($column, $type) {
+	public function castColumn(string $column, $type): string {
```

Comment:
Same thing here, needs it be:
```suggestion
	public function castColumn(string $column, $type): IQueryFunction {
```
Otherwie the QuoteHelper breaks when you try to cast a column to string/int before using it in any other expression (which is it's single usecase)

Reply:
But then we need to always construct a IQUeryFunction on the cast. Our types have to be consistent. Else it is just asking for trouble.